lobby.cpp

```diff
@@ -331,11 +330,16 @@ void Lobby::on_chatfield_return_pressed()
 
   AOPacket *f_packet = new AOPacket(f_header, f_contents);
 
-  ao_app->send_ms_packet(f_packet);
+  send_ms_packet(f_packet);
 
   ui_chatmessage->clear();
 }
 
+void Lobby::on_loading_cancelled()
+{
+  cancel_clicked();
```

Comment:
```suggestion
  emit cancel_clicked();
```

Although you might want to consider eliminating this slot and connecting the signal directly.

Reply:
Yeah, probably.
Also, I've seen a few more `emit`-less signals, hence why I did this. So I'll go ahead and give those an `emit` as well.